sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -128,6 +128,13 @@ def __init__(self, body, **kwargs):
     def __str__(self):
         return str(self.message)
 
+    def __repr__(self):
+        # type: () -> str
+        rec_repr = "body={}".format(
+            self.body
```

Comment:
also wondering if other public properties should be included, it seems like EH is exposing all the public properties:

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/azure/eventhub/_common.py#L104-L132